src/routes/safe/components/Transactions/TxList/TxDataRow.tsx

```diff
@@ -0,0 +1,41 @@
+import { ReactElement, ReactNode } from 'react'
+import styled from 'styled-components'
+import { CopyToClipboardBtn, Text } from '@gnosis.pm/safe-react-components'
+
+import { InlineEthHashInfo, StyledGridRow } from './styled'
+import { getExplorerInfo } from 'src/config'
+import { getByteLength } from 'src/utils/getByteLength'
+
+const FlexWrapper = styled.div<{ margin: number }>`
+  display: flex;
+  align-items: center;
+
+  > :nth-child(2) {
+    margin-left: ${({ margin }) => margin}px;
+  }
+`
+
+type TxDataRowType = { children?: ReactNode; inlineType?: 'hash' | 'rawData'; title: string; value: string | null }
```

Comment:
Why don't you just make `value` optional? You won't need to check the type each time.